api/apps/geoprocessing/src/modules/surface-cost/ports/shapefile-record.ts

```diff
@@ -0,0 +1,4 @@
+export interface ShapefileRecord {
```

Comment:
very minor - could we rename this to stick more closely to the specific record kind? as is now, its name is quite generic (I know, it's within the `surface-cost` module, but it's not strictly speaking namespaced)

Reply:
```suggestion
export interface CostSurfaceShapefileRecord {
```
maybe?

Reply:
Sure!